manifests/supervisorcluster/1.22/cns-csi.yaml

```diff
@@ -515,6 +515,11 @@ webhooks:
         operations:  ["CREATE", "UPDATE", "DELETE"]
         resources:   ["persistentvolumeclaims"]
         scope: "Namespaced"
+     - apiGroups:   ["snapshot.storage.k8s.io"]
```

Comment:
@vdkotkar @divyenpatel The format of the yaml is incorrect. This broke the pipeline as we are unable to push this commit. Can we fix this please?

Reply:
@divyenpatel can we have the yaml lint check added by default so that we can catch such issues before merge? 

yamllint - https://yamllint.readthedocs.io/en/stable/

Reply:
PR is out to fix the issue - https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/2315